mood_tracker/models.py

```diff
@@ -24,15 +25,20 @@ class Achievements(models.Model):
     achievement_icon = models.ImageField(
         upload_to='images/', default='../default-pic_ls0v0g.png')
     achievement_description = models.CharField(max_length=250)
+    achievement_user = models.ForeignKey(User, related_name='achievements', blank=True, on_delete=models.CASCADE)
+    
+    def __str__(self):
+        return self.achievement_name
 
 
 class Diary(models.Model):
     title = models.CharField(max_length=200)
     content = models.TextField()
     date_created = models.DateTimeField(default=timezone.now)
-
+    owner = models.ForeignKey(User, related_name='diaries', blank=True, on_delete=models.CASCADE)
```

Comment:
Have you updated your codebase as there were things that were just added to the model about 15 min ago.

Reply:
Yes, I edited the models